src/app/components/article/ArticleForm.js

```diff
@@ -35,7 +36,7 @@ const ArticleForm = ({
 
   const [claimDescription, setClaimDescription] = React.useState(articleType === 'fact check' && projectMedia?.suggested_main_item ? projectMedia.suggested_main_item.claim_description : '');
```

Comment:
`fact-check` to match propType

Reply:
I thought about this too, but I didn't want to change too much in the forms in the scope of this ticket since @sarahkeh is still working on them.

Reply:
This is something I also did in my branch, so we're doubly covered

Reply:
Awesome, thanks @sarahkeh !

Reply:
Awesome! :bananadance: